web-client/src/app/views/delete-user/delete-user.page.ts

```diff
@@ -40,18 +44,47 @@ export class DeleteUserPage implements OnInit {
   /** @see validatedAddress */
   address?: string;
 
+  hideXrpBalance = environment.hideXrpBalance;
```

Comment:
can just use environment.hideXrpBalance as is, don't need to create separate variable

Reply:
I created a separate variable to use `hideXrpBalance` in [delete-user.page.html](https://github.com/ntls-io/nautilus-wallet/blob/aaef347fe516a445a01d9ed6e577beed5afd5441/web-client/src/app/views/delete-user/delete-user.page.html#L10). I tried using `environment.hideXrplBalance` inside the ngIf directive, but it looks like I need to initialize the variable inside delete-user.page.ts.